src/cli.ts

```diff
@@ -52,13 +55,24 @@ export class CliChannel {
         return optsCopy;
     }
 
-    async executeTool(command: CommandText, opts?: cp.ExecOptions, fail = true): Promise<CliExitData> {
+    async executeTool(command: CommandText, opts?: cp.ExecOptions, fail = true, executionContext?: ExecutionContext): Promise<CliExitData> {
         const commandActual = command.toString();
         const commandPrivacy = command.privacyMode(true).toString();
         const [cmd] = commandActual.split(' ');
         const toolLocation = await ToolsConfig.detect(cmd);
         const optsCopy = CliChannel.applyEnv(opts, CliChannel.createTelemetryEnv())
+
+        const key = `${command.command} ${[command.parameter, ...command.options.map((o)=>o.toString())]}`;
```

Comment:
I believe that `commandActual` is the same string as this `key`, and that has already been computed

Reply:
It's not the same, but actually using `commandActual` value is better.
Fixed